### PR TITLE
Add installation instructions for other OSes

### PR DESCRIPTION
**Pull request by chrissnell**
Break out separate installation instructions for Windows/Mac and Linux.  Add link to Arch AUR repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,12 +6,15 @@ Inter UI is a typeface specially designed for user interfaces.
 
 ### [⬇︎ Download the latest release](https://github.com/rsms/inter/releases)
 
+### Windows and Mac Users
```
Comment (rsms) [inline README.md]: Could you drop this?
(99.9% of people downloading Inter UI will be on Windows or Mac.)

File: README.md [hunk 2]
```diff
@@ -6,12 +6,15 @@ Inter UI is a typeface specially designed for user interfaces.
...
 After downloading the zip from above:
 
 1. Double-click the downloaded zip file to unpack or open it.
 2. Follow the instructions in "install-mac.txt" or "install-win.txt", depending
    on what operating system you're using.
 
+### Linux Users
```
Comment (rsms) [inline README.md]: s/Linux Users/Linux/

File: README.md [hunk 3]
```diff
@@ -6,12 +6,15 @@ Inter UI is a typeface specially designed for user interfaces.
...
+For Arch Linux users, [a package is available in the AUR](https://aur.archlinux.org/packages/ttf-inter-ui/).
```
Comment (rsms) [inline README.md]: Please add general instructions for Linux (I'm only using Linux without a window manager, so I'm not familiar with font management in various dists; perhaps there's no unified way. If so, please add a note pointing that out or somehow make it clear to other Linux users that they might have to go someplace else to learn how to install fonts.)

**Comment by rsms** [comment]
Ping!

File: README.md (hunk 3 above)
Comment (chrissnell) [reply README.md]: I don't know what general instructions for Linux look like.  Almost every distribution is different here.   Feel free to close out this PR.

**Comment by rsms** [comment]
Closing dead PR

**Pull request closed without merging**